redbeams/src/main/java/com/sequenceiq/redbeams/RedbeamsApplication.java

```diff
@@ -57,7 +58,9 @@
         "com.sequenceiq.cloudbreak.ha.service",
         "com.sequenceiq.cloudbreak.tracing",
         "com.sequenceiq.cloudbreak.tag",
-        "com.sequenceiq.cloudbreak.filter"},
+        "com.sequenceiq.cloudbreak.filter",
+        "com.sequenceiq.cloudbreak.common"
```

Comment:
adding every package by hand seems a bit painful, what's the reason behind not adding `com.sequenceiq` ?

Reply:
This is how redbeams was started. The goal was to parse only what is needed and nothing more.

Reply:
not a good excuse :) we should consider changing this, I don't think it makes too much sense in this form. If someone is not familiar enough with redbeams can spend a lot of time figuring out what's wrong before gets to this point

Reply:
Ok. However, I would do it in a follow-up, as this is not the scope of this PR.

Reply:
https://jira.cloudera.com/browse/CB-9323